gnd/src/main/java/com/google/android/gnd/model/Project.java

```diff
@@ -39,6 +40,9 @@
   @NonNull
   protected abstract ImmutableMap<String, Layer> getLayerMap();
 
+  @NonNull
+  public abstract ImmutableList<OfflineBaseMapSource> getOfflineBaseMapSources();
```

Comment:
Cool! What's the envisioned flow for selecting between these? 

Offline Areas -> Add Area -> Select Source -> Download... or something different?

Reply:
These will be configured via the web app only - there will likely be an overflow menu with advanced project settings where we can paste the URL.